app/src/components/governance/ProposalDescription.vue

```diff
@@ -1,14 +1,20 @@
 <template>
   <div id="proposal-description" class="proposal-description">
     <div class="description-content-container">
-      <section v-if="description">
+      <section v-if="proposal && proposal.description">
         <h4>Description</h4>
         <div class="description">
-          <pre v-if="type === `PARAMETER_CHANGE` || type === `TREASURY`">
-          {{ description }}
-          <p class="parameter">New Parameter: {{ parameter }}</p>
+          <pre
+            v-if="
+              proposal.type === `PARAMETER_CHANGE` ||
+              proposal.type === `TREASURY`
+            "
+          >
+          {{ proposal.description }}
+          <p class="parameter">New Parameter: {{ proposal.changes }}</p>
```

Comment:
instead let's create the description on the server?

Reply:
?? what do you mean?

Reply:
create the description on the server and not send "parameter" and "beneficiary"